### DISCUSSION TITLE
tutorials/intro-dgs/03-schema-definition-language-sdl

### DISCUSSION THREAD

**Discussion by giscus** [apollographql-education/content-feedback#103] [course feedback]
# tutorials/intro-dgs/03-schema-definition-language-sdl

Live data, resolvers, and data sources

https://www.apollographql.com/tutorials/intro-dgs/03-schema-definition-language-sdl

**Comment by mkatsman** [comment]
this is written before building the schema and creating the directory.
 "(specifically, by looking in the project's src/main/resources/schema folder, which we just created),"

It is confusing. I looked for that dir for a while.

**Reply by lizhennessy** [reply]
Thank you so much for calling this out @mkatsman . A fix is on the way!
